swap_meet/item.py

```
@@ -1,2 +1,37 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        if id is  None:
+            self.id =  uuid.uuid4().int 
+        else:
+            self.id = id 
+        
+        if isinstance(condition, (int, float)):
```

Validating arguments's values is a generally good idea! If another developer tries to create an instance of `Item` with something other than an `int` or `float`, this will cause the `TypeError` to be raised. It's a good idea to consider the implications of raising an exception though. Rhetorically speaking, do you want to potentially halt the execution of the program if raising this exception is not handled elsewhere? I suspect not, but definitely not universally the case.  Another option might be to cast the value to something else if it is the wrong type. You could use your `isinstance` check and if it fails set the condition to `None` or `0` or something.